nulinks-common/test/__snapshots__/data.search.test.js.snap

```diff
@@ -80,13 +84,13 @@ exports[`searching data.js matches snapshot for term s 1`] = `"Service Time Logs
 
 exports[`searching data.js matches snapshot for term sched 1`] = `"My Schedule"`;
 
-exports[`searching data.js matches snapshot for term t 1`] = `"Access XFINITY On-Campus"`;
+exports[`searching data.js matches snapshot for term t 1`] = `"Microsoft Office 365"`;
 
 exports[`searching data.js matches snapshot for term u 1`] = `"myHealth Center (UHCS)"`;
 
 exports[`searching data.js matches snapshot for term v 1`] = `"On The Hub"`;
 
-exports[`searching data.js matches snapshot for term w 1`] = `"Student Employment (workstudy, timesheets)"`;
+exports[`searching data.js matches snapshot for term w 1`] = `"Microsoft Office 365"`;
```

Comment:
This one I'm not a huge fan of either. I feel like most people use `word` offline so they would maybe need it once, but they might want to search from `workstudy` more than just the once. I don't use either, so I don't have a great pulse on it.